build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
Maybe I should bite the bullet and just call it `JAVA_HOME`, although I'm a little worried it might interfere with system packaged Java apps that we are using, like ant. Thoughts?